src/rofl/datapath/pipeline/openflow/openflow1x/pipeline/of1x_flow_table.h

```diff
@@ -172,6 +172,13 @@ rofl_result_t __of1x_destroy_table(of1x_flow_table_t* table);
 */
 
 /**
+* @internal Add flow entry to table being able to match cookie
+*
+* @warning Out of spec.
```

Comment:
Which documentation do you mean?

To the second question, yes exactly; you will endup with "backup" entries, which are actually never matched. That's precisely the kind of rules that OF, per se, doesn't allow you to install, but that through this alternative API you can potentially install